src/components/App.js

```diff
@@ -5,10 +5,21 @@ import '../css/styles.css';
 
 class App extends React.Component {
 
+  apiCheck() {
```

Comment:
Here's 2 options to do this better: 
1) (preferred but difficult) useEffect hook
2) Create a file `utils/fetch` that has this fetch code and accepts as parameters the url, body, verb etc. that way you're abstracting the 3rd party you're using to only be called in one place, making it easier to remove/update later.